Data-Engineering/Airflow/example_ezaf_spark_csv_to_parquet_fts.yaml

```diff
@@ -4,37 +4,36 @@ metadata:
   name: spark-fts-{{ts_nodash|replace("T", "")}}
 spec:
   sparkConf:
-    spark.mapr.extraconf.secret: {{dag_run.conf.get("s3_secret_name", "spark-s3-creds")}}
```

Comment:
Why did you decide to go with passing creds via sparkConf instead of using 'spark.mapr.extraconf.secret'? I believe previous comment from @akravacyber is also related to this.
If you use spark.mapr.extraconf.secret - you don't need to set access key and secret key explicitly in the yaml.

Reply:
Agree, I think it's better to use secret, rather than directly pass jwt token. Secret can be manually created from the kubeflow notebook

Reply:
Spark S3 secret notebook is already present. Please refer https://github.com/HPEEzmeral/ezua-tutorials/tree/release/fy23-q4/Data-Analytics/Spark-S3-secret